ecosystem/sep-0006.md

```diff
@@ -69,6 +71,12 @@ Access-Control-Allow-Origin: *
 
 This protocol involves the transfer of value, and so HTTPS is required for all endpoints for security.  Wallets and anchors should refuse to interact with any insecure HTTP endpoints.
 
+## Asset Conversions
+
+This protocol was originally designed to provide conversions between off-chain and on-chain equivalent assets – for instance `BRL <> BRLT`, `USD <> USDC`, `NGN <> NGNT`, etc. – but it also describes how to make conversions between off-chain and on-chain non-equivalent assets like `BRL <> USDC`.
+
+To enable the conversion of non-equivalent assets the Anchor needs to: (i) mark the asset with `"quotes_supported": true` in its [`/info`](#info) endpoint; and (ii) implement the [SEP-38 Anchor RFQ API](sep-0038.md), which enables the Anchor to provide quotes to the Wallet. These quotes can be [indicative](https://www.investopedia.com/terms/i/indicativequote.asp) or [firm](https://www.investopedia.com/terms/f/firmquote.asp). If a firm quote is used, the Stellar transaction _must_ be submitted to the Stellar network prior to the quote's expiration. More specifically, the `created_at` timestamp returned in the [Horizon GET /transactions/:id](https://developers.stellar.org/api/resources/transactions/) request must be earlier than the quote's expiration.
```

Comment:
💡  I think it would be more helpful to outline the flow of funds (user --> anchor --> user) for both the deposit and withdrawal case where non-equivalent assets are being exchanged, instead of listing the technical requirements in the same section that introduces the concept.

The reason differentiating between deposit and withdrawal is necessary is because in the deposit case the user has to take steps to deliver funds to the anchor's off-chain account before the quotes expiration. In the withdrawal case the wallet just needs to send funds from the user's Stellar account before the quotes expiration.

I also think we can leave the details of listing `quotes_supported` outside this section.

Reply:
Cool, I like those suggestions a lot! Take a look if this updated description would make things clearer: https://github.com/stellar/stellar-protocol/pull/973/commits/66d8220a12e8325fbe523f0f632820a79ef278b5